docs/package.json

```diff
@@ -90,11 +101,10 @@
   },
   "ember-addon": {
     "paths": [
-      ".."
+      "../addon"
```

Comment:
I think we can drop that! IIRC that's for in-repo addons before we had monorepos, wasn't it? Anyways, we should add the addon as a devDependency and let pnpm do the cross linking:

```
"ember-bootstrap": "workspace:*".
```